components/authz-service/server/v2/project_test.go

```diff
@@ -139,6 +140,25 @@ func TestCreateProject(t *testing.T) {
 			grpctest.AssertCode(t, codes.AlreadyExists, err)
 			assert.Nil(t, resp)
 		}},
+		{"does not create project if max projects allow surpassed", func(t *testing.T) {
```

Comment:
Either "allowed" or just drop the word; it retains the same meaning.
Same applies in postgres_test.go